modules/core/src/main/java/org/apache/synapse/ServerContextInformation.java

```diff
@@ -1,152 +1,162 @@
+                                    ServerConfigurationInformation serverConfigurationInformation) {
+        this.serverContext = serverContext;
+        this.serverConfigurationInformation = serverConfigurationInformation;
+    }
+
+    public Object getServerContext() {
+        return serverContext;
+    }
+
+    public void setServerContext(Object serverContext) {
+        this.serverContext = serverContext;
+    }
+
+    public void addProperty(String key, Object value) {
+        properties.put(key, value);
+    }
+
+    public Object getProperty(String key) {
+        return properties.get(key);
+    }
+
+    public ServerState getServerState() {
+        return serverState;
+    }
+
+    public void setServerState(ServerState serverState) {
+        this.serverState = serverState;
+    }
+
+    public SynapseConfiguration getSynapseConfiguration() {
+        return synapseConfiguration;
+    }
+
+    public void setSynapseConfiguration(SynapseConfiguration synapseConfiguration) {
+        this.synapseConfiguration = synapseConfiguration;
+    }
+
+    public SynapseEnvironment getSynapseEnvironment() {
+        return synapseEnvironment;
+    }
+
+    public void setSynapseEnvironment(SynapseEnvironment synapseEnvironment) {
+        this.synapseEnvironment = synapseEnvironment;
+    }
+
+    public SynapseCallbackReceiver getSynapseCallbackReceiver() {
+        return synapseCallbackReceiver;
+    }
+
+    public void setSynapseCallbackReceiver(SynapseCallbackReceiver synapseCallbackReceiver) {
+        this.synapseCallbackReceiver = synapseCallbackReceiver;
+    }
+
+    public ServerConfigurationInformation getServerConfigurationInformation() {
+        return serverConfigurationInformation;
+    }
+
+    /**
+     * Returns the number of current callbacks.
+     *
+     * @return the number of current callbacks.
+     */
+    public int getCallbackCount() {
+        if (synapseCallbackReceiver != null) {
+            return synapseCallbackReceiver.getCallbackCount();
+        }
+        return 0;
+    }
+
+    public SynapseDebugManager getSynapseDebugManager(){
+        return debugManager;
+    }
+
+    public void setSynapseDebugManager(SynapseDebugManager debugManager){this.debugManager=debugManager;}
+
+    public SynapseDebugInterface getSynapseDebugInterface(){
+        return debugInterface;
+    }
+
+    public void setSynapseDebugInterface(SynapseDebugInterface debugInterface) {
+        this.debugInterface = debugInterface;
+    }
+
+    public boolean isServerDebugModeEnabled(){
+        return isDebugModeEnabled;
+    }
+
+    public boolean isServerUnitTestModeEnabled(){
+        return isUnitTestModeEnabled;
+    }
+
+    public void setServerDebugModeEnabled(boolean isDebugModeEnabled){
+        this.isDebugModeEnabled=isDebugModeEnabled;
+    }
+
+    public void setServerUnitTestModeEnabled(boolean isUnitTestModeEnabled){
+        this.isUnitTestModeEnabled=isUnitTestModeEnabled;
```

Comment:
Shall we format only the newly added lines, seems the whole class is formatted

Reply:
Done